### PR TITLE
Enable Google chart for profile plugin.

### PR DESCRIPTION
**Pull request by qiuminxu**
Add Google Chart Webcomponents from
https://github.com/GoogleWebComponents/google-chart
Also added a demo to use Google Chart.

google_chart_demo

### CODE REVIEW

File: tensorboard/plugins/profile/google_chart/google-chart.html [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+      }
+      dataPromise
+        .then(this.$.loader.dataTable.bind(this.$.loader))
+        .then(this.$.loader.dataView.bind(this.$.loader))
+        .then(function(dataView) {
+          this._dataView = dataView;
+        }.bind(this));
+    },
+
+    attached: function() {
```
Comment (jart) [inline tensorboard/plugins/profile/google_chart/google-chart.html]: Thank you!

File: tensorboard/plugins/profile/profile_demo.py [hunk 2]
```diff
@@ -65,6 +65,9 @@ def dump_data(logdir):
     shutil.copyfile(
         'tensorboard/plugins/profile/profile_demo.memory_viewer.json',
         os.path.join(run_dir, 'memory_viewer.json'))
+    shutil.copyfile(
```
Comment (jart) [inline tensorboard/plugins/profile/profile_demo.py]:
```diff
Author: Justine Tunney <jart@google.com>
Date:   Mon Jul 9 09:32:09 2018 -0700

    Add missing data deps for profile_demo

diff --git a/tensorboard/plugins/profile/BUILD b/tensorboard/plugins/profile/BUILD
index bff97bee..dea41737 100644
--- a/tensorboard/plugins/profile/BUILD
+++ b/tensorboard/plugins/profile/BUILD
@@ -73,7 +73,11 @@ py_binary(
         "profile_demo.py",
         "profile_demo_data.py",
     ],
-    data = ["profile_demo.op_profile.json"],
+    data = [
+        "profile_demo.google_chart_demo.json",
+        "profile_demo.memory_viewer.json",
+        "profile_demo.op_profile.json",
+    ],
     srcs_version = "PY2AND3",
     deps = [
         ":profile_plugin",
```

File: tensorboard/plugins/profile/google_chart/google-chart.html (hunk 1 above)
Comment (qiuminxu) [reply tensorboard/plugins/profile/google_chart/google-chart.html]: Done. Thanks!

File: tensorboard/plugins/profile/profile_demo.py (hunk 2 above)
Comment (qiuminxu) [reply tensorboard/plugins/profile/profile_demo.py]: Done. Thanks!

**Pull request merged**